docs/tutorial/part-zero/index.md

```diff
@@ -25,15 +25,34 @@ Take a moment to locate and open up the command line interface (CLI) for your co
 
 > 💡 For a great introduction to using the command line, check out [**Codecademy’s Command Line tutorial**](https://www.codecademy.com/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command) for Mac and Linux users, and [**this tutorial**](https://www.computerhope.com/issues/chusedos.htm) for Windows users. Even if you are a Windows user, the first page of the Codecademy tutorial is a valuable read. It explains what the command line is, not just how to interface with it.
 
-## Install Node.js
+## Install Homebrew for Node.js
+
+To install Gatsby and Node.js, it is recommended to use [Homebrew](https://brew.sh/). A little set-up in the beginning can save you from some headaches later on!
+
+How to install or verify Homebrew on your computer:
+
+1. Open your Terminal
+1. See if Homebrew is installed by running `brew -v`. You should see "Homebrew" and a version number, if so.
```

Comment:
the `if so` reads a bit strange in my opinion, it looks like something is missing.
Wouldn't it be better to just omit that?

Reply:
Makes sense to me, I've updated the PR!